llvm-runtimes/compiler-rt/compiler-rt-21.0.0.9999.ebuild

```diff
@@ -63,7 +63,7 @@ test_compiler() {
 }
 
 src_configure() {
-	llvm_prepend_path "${LLVM_MAJOR}"
+	llvm_prepend_path -b "${LLVM_MAJOR}"
```

Comment:
We do have a `DEPEND` on LLVM here, so this change is likely wrong, or at least wrong by itself. You probably need `-DLLVM_ROOT` like I've started adding in other packages.

Reply:
Nice, I think using `-DLLVM_ROOT` and removing `llvm_prepend_path` is going to solve my issue, but let me test it carefully today.

Reply:
I confirm that it works! I'm able to bootstrap the whole cross toolchain with just `-DLLVM_ROOT`.